tests/test_ui_tabs_enhanced.py

```
@@ -0,0 +1,256 @@
+
+    async with app.run_test() as pilot:
+        from textual.widgets import TabbedContent, DataTable
+
+        tabbed = pilot.app.query_one(TabbedContent)
+        tabbed.active = "jobs"
+        await pilot.pause()
+
+        try:
+            jobs_table = pilot.app.query_one("#jobs-table-full", DataTable)
+            assert jobs_table is not None
+        except:
```

Ruff (0.14.14)

Do not use bare `except`

(E722)